src/main/java/model/item/Equipment.java

```diff
@@ -1,50 +1,56 @@
 package model.item;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
+import javafx.beans.property.IntegerProperty;
+import javafx.beans.property.ReadOnlyIntegerProperty;
+import javafx.beans.property.SimpleIntegerProperty;
 import manager.Database;
 import manager.TypTranslation;
+import manager.Utility;
 import model.Currency;
 import model.upgrade.Upgrade;
 import model.upgrade.UpgradeFactory;
 
 import java.util.*;
+import java.util.function.Consumer;
 import java.util.stream.Collectors;
 
 public abstract class Equipment extends Item {
```

Comment:
Equipment is getting pretty large :) 
Maybe it is time to add interfaces like Breakable for wear, Upgradable, ...

Reply:
The problem is that Equipment wouldn't get smaller.
There is no multiple inheritance in Java :(